openadapt/utils.py

```diff
@@ -48,7 +51,9 @@
 from openadapt.custom_logger import filter_log_messages
 from openadapt.db import db
 from openadapt.models import ActionEvent
+from config import Config
 
+config = Config()
```

Comment:
This should not be necessary. Please replace with `from openadapt.config import config` like it's used elsewhere.